shared/ssa/codeql/ssa/Ssa.qll

```diff
@@ -1599,6 +1599,13 @@ module Make<LocationSig Location, InputSig<Location> Input> {
       /** Gets the basic block to which this node belongs. */
       abstract BasicBlock getBasicBlock();
 
+      /**
+       * INTERNAL: Do not use.
+       *
+       * Gets the basic block index of this node.
+       */
+      abstract int getIndex();
```

Comment:
This also needs to be implemented for the other languages, right? Otherwise, I assume compilation will fail?